examples/pipelines/hf_dataset_pipeline/components/load_from_hub/README.md

```diff
@@ -0,0 +1,14 @@
+This component can be run as follows:
```

Comment:
It documents the arguments specific to this component, so for me this is fine to have here.

Reply:
I think it's more general to data loading in general which will be similar for all dataset frameworks. IMO it should be only mentioned once in a reference doc. I think it would be nice to have a description of the component as well as a schema description for the args in `--extra-args`